### PR TITLE
Example fixes

### PR DESCRIPTION
**Pull request by simonduq**

### CODE REVIEW

File: examples/6tisch/6p-packet/sixp-node.c [hunk 1]
```diff
@@ -49,7 +51,7 @@ PROCESS_THREAD(sixp_node_process, ev, data)
 
   sixtop_add_sf(&test_sf);
 
-  if(node_id == COORDINATOR_NODE_ID) {
+  if(linkaddr_cmp(&coordinator_addr, &linkaddr_node_addr)) {
```
Comment (atiselsts) [inline examples/6tisch/6p-packet/sixp-node.c:54]: Are you sure this keeps the original intent? I mean, `node_id` can equal to `1` for other MAC addresses, not only this one.

Comment (simonduq) [reply examples/6tisch/6p-packet/sixp-node.c:54]: Yes I'd say the original intent was just to hardcode a coordinator.

**Comment by simonduq** [comment]
NB: this is still WIP, more commits coming in :)

**Comment by simonduq** [comment]
OK that's it for now feel free to merge :)

**Comment by simonduq** [comment]
OK now with the fix to arm-01 compile test

**Comment by atiselsts** [comment]
Separated in two parts - one goes with the build-all.sh PR #311, the other with actual fixes in a new PR #324

**Pull request closed without merging**